manifests/charts/istio-control/istio-discovery/templates/mutatingwebhook.yaml

```diff
@@ -37,10 +37,17 @@ webhooks:
     namespaceSelector:
 {{- if .Values.sidecarInjectorWebhook.enableNamespacesByDefault }}
       matchExpressions:
+      {{- if .Values.sidecarInjectorWebhook.excludeNamespaces }}
+      - key: name
```

Comment:
Just to be sure this is known, `name` is NOT the name of the namespace. It is a `name` label, which is not automatically added (and doesn't follow k8s naming practices!). There is some KEP to automatically add a similar label, but its not ready yet

Reply:
Thanks for the info @howardjohn. This is good to know. Where is the KEP for this?